### PR TITLE
Fix failed to parse sourcemap (issue #396)

### PR DESCRIPTION
**Pull request by alvin883**
I've encountered this issue: https://github.com/mebjas/html5-qrcode/issues/396, because the `src` folder was not included to the published package on npm thus it caused a bunch of warnings, so I include the `src` with this PR

For the `third_party` folder, I'm not sure where you got the file so I remove the sourcemap instead to avoid any ENOENT warning from `zxing-js`'s sourcemap because `third_party/index.js.map` is not exist anywhere

#### Before
image

#### After
image

PS:
I saw from [this readme](https://github.com/mebjas/html5-qrcode/blob/master/minified/readme.md) that minified version is deprecated, however the last changes for that file is last month, therefore I push the changes for the minified version as well, here if you want to see the diff check of `minified/html5-qrcode.min.js`
https://www.diffchecker.com/W8xqZDZd/ (not sure if you need it, just in case :sweat_smile: )

Thank you!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,8 +26,8 @@
     "build:typing": "tsc --emitDeclarationOnly --outDir ./dist",
     "build:umd": "./scripts/build-webpack.sh",
     "build:umd_windows": ".\\scripts\\build-webpack.bat",
-    "build:copy": "cp README.md dist && cp package.json dist && cp LICENSE dist",
-    "build:copy_windows": "copy README.md dist && copy package.json dist && copy LICENSE dist",
+    "build:copy": "cp README.md dist && cp package.json dist && cp LICENSE dist && cp -R src dist/src",
+    "build:copy_windows": "copy README.md dist && copy package.json dist && copy LICENSE dist && xcopy .\\src .\\dist\\src",
```
Comment (Swerner1975) [inline package.json:30]: "build:copy_windows": "copy README.md dist && copy package.json dist && copy LICENSE dist && xcopy .\\src .\\dist\\src  **/S/I/Y**"

benutzen, sonst kopiert er einige Unterverzeichnisse nicht mit. Ausserdem kommt sonst immer eine Abfrage nach Überschreiben.

**Comment by mebjas** [comment]
Thanks for the PR.

Please resync the code -- happy to review and merge.

File: package.json (hunk 1 above)
Comment (alvin883) [reply package.json]: updated, thank you I learned new thing .. and German language 😅

**Review by mebjas — APPROVED** [review]
Everything else looks good to me.

**Comment by alvin883** [comment]
updated, thank you!

**Comment by mebjas** [comment]
Thanks for the PR, merging!

**Comment by Swerner1975** [comment]
See the warnings in the New Release 2.3.8 too...
I have hoped the problem is fixed...

**Comment by jean-webdev** [comment]
Same, seeing warnings in 2.3.8. Looking forward to a fix, because it's otherwise a great library and I'd rather not disable source mapping

**Pull request merged**